service/vspo-schedule/web/src/components/Head/Head.tsx

```diff
@@ -23,7 +23,7 @@ export const CustomHead = ({
       <meta property="og:description" content={description} />
       <meta
         property="og:url"
-        content={`https://www.vspo-schedule.com/${path}`}
+        content={`https://www.vspo-schedule.com${path}`}
```

Comment:
I am not sure but maybe we could use related path here so that if we want to change domain we won't need to rewrite the url again?

Reply:
It seems that the `og:url` must be entered as an absolute path, making it difficult to use a relative path. (https://ogp.me/)

I will set the environment variables on the Vercel side myself.
If you let me know what environment variables to add, I will add them before deploying🙆‍♂️.

Extra Notes:

However, when wanting to check the OG in Preview, it's challenging to confirm if `og:url` and others are functioning correctly at the Preview stage since the Origin varies with each Preview. Using environment variables makes this difficult.
Well, if using PageRouter and `getServerSideProps`, it might be possible to construct the meta tag in CustomHead by obtaining the URL of the request included in the context... But I'm struggling to find a solution for when wanting to use SSG. 🫠